test-e2e/docker_smoke_test.go

```diff
@@ -0,0 +1,144 @@
+package test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+const (
+	testDir = "waypoint-examples/docker/go"
```

Comment:
should we make the path to the waypoint-examples repo a test env var as well? that way i can smoke test against a specific branch or somesuch that i already have cloned/checked out.

Reply:
Ooh good idea! 👍